src/Bridge/Doctrine/Orm/Extension/EagerLoadingExtension.php

```diff
@@ -66,18 +82,20 @@ public function getMetadataProperties(string $resourceClass) : array
      *
      * @param QueryBuilder $queryBuilder
      * @param string       $resourceClass
+     * @param array        $context       the serialization context
```

Comment:
+1 fixed in #803
IMO we should think about giving the full context through dataproviders and extensions instead of the operation_ame. Would make things easier in some of them, and also remove some code (everytime we build the context again for some purpose).